src/bundlingCalls/bundlingUtils.ts

```diff
@@ -38,8 +37,14 @@ export function computeBundleId(
 ) {
   const ids: unknown[] = [];
   let hasIds = false;
-  for (let i = 0; i < discriminatorFields.length; ++i) {
-    const id = at(obj, discriminatorFields[i])[0];
+  for (const field of discriminatorFields) {
+    const pathParts = field.split('.');
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let currentObj: any = obj;
+    for (const pathPart of pathParts) {
+      currentObj = currentObj[pathPart];
```

Comment:
I believe this would throw on a path that doesn't exist due to accessing a property on undefined.

You could replace with `currentObj?[]` I believe.